python/cudf_polars/cudf_polars/experimental/benchmarks/pdsh.py

```diff
@@ -25,13 +25,17 @@
 from typing import TYPE_CHECKING, Any
 
 import numpy as np
-import pynvml
 
 import polars as pl
 
-from cudf_polars.dsl.translate import Translator
-from cudf_polars.experimental.explain import explain_query
-from cudf_polars.experimental.parallel import evaluate_streaming
+try:
+    import pynvml
+
+    from cudf_polars.dsl.translate import Translator
+    from cudf_polars.experimental.explain import explain_query
+    from cudf_polars.experimental.parallel import evaluate_streaming
+except ImportError:
+    print("Assume CPU execution...")
```

Comment:
I think we should separate this a little. But this is fine to do in a follow up.
```python
try:
    import pynvml
except:
    pynvml = None

try:
    from cudf_polars.dsl.translate import Translator
    from cudf_polars.experimental.explain import explain_query
    from cudf_polars.experimental.parallel import evaluate_streaming
    CUDF_POLARS_AVAILABLE = True
except ImportError:
    CUDF_POLARS_AVAILABLE = False
```

And then guard uses with `if pynvml is None` and `if CUDF_POLARS_AVAILABLE`

Reply:
Good idea.  I'll update the PR